### PR TITLE
KogeCoin

### PR DESCRIPTION
**Pull request by loganbek**
#### Hi! What is your PR about?

- [x] Add or edit a skin
- [ ] Add custom domain for your space
- [ ] Add alias for migrate your space
- ~Add images for your space~ (deprecated) Add the image URL on the space settings directly
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: skins/kogecoin.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    --primary-color: #41218e;
+    --bg-color: ##eaeaea;
+    --text-color: #000;
+    --link-color: #0c6eea;
+    --heading-color: #ffffff;
+    --border-color: #ffffff;
+    --header-bg: #fba130;
+    --block-bg: #41218e;
+  }
+
```
Comment (gokaiorg) [inline skins/kogecoin.scss:11]: image

- The colours used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
- Please delete the extra line.

```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
We have a new method to insert your images without creating a Pull Request. You can now use the Avatar field as your space settings. Please remove them from your PR.
image

**Comment by loganbek** [comment]
@gokaiorg - I removed the logos and updated the CSS. Thanks!

File: spaces/tokens.json [hunk 2]
```diff
@@ -152,5 +152,7 @@
   "0xeaccb6E0f24d66cF4Aa6cBDa33971b9231d332a1": "polyient-games-governance",
   "0xaea8e1b6cb5c05d1dac618551c76bcd578ea3524": "sogur",
   "0x683EcFe5B148c8840998c1B4884F60a8eAD432D1": "d5",
-  "0xa150db9b1fa65b44799d4dd949d922c0a33ee606": "digital-reserve-currency"
+  "0xa150db9b1fa65b44799d4dd949d922c0a33ee606": "digital-reserve-currency",
+  "0x13748d548d95d78a3c83fe3f32604b4796cffa23": "kogecoin"
+
```
Comment (gokaiorg) [inline spaces/tokens.json:157]: Please do not add your token to this file, you don't need it.
```suggestion
  "0xa150db9b1fa65b44799d4dd949d922c0a33ee606": "digital-reserve-currency"
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/tokens.json (hunk 2 above)
Comment (loganbek) [reply spaces/tokens.json]: Removed from tokens.json.

**Comment by loganbek** [comment]
@gokaiorg let me know if you need anything else changed. Thanks.

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**